.circleci/config.yml

```diff
@@ -0,0 +1,89 @@
+version: 2.1  # The version of CircleCI platform being used
+executors:
+  python-docker:
+    working_directory: ~/repo
+    docker:
+      - image: cimg/python:3.6
+
+
+jobs:  # The jobs level contains a collection of arbitrarily named children
+  run-tests:
+    executor: python-docker
+    steps:  # The steps collection is an ordered list of run directives
+      - checkout  # "checkout" checks out the source code to "working_directory"
+      - restore_cache:
+          key: pip-deps-v1-{{ checksum "constraints.txt" }}-{{ checksum "setup.py" }}-{{ arch }}
+      - run:
+          name: Install Python Dependencies
+          command: |
+            set -euxo pipefail
+            python -m venv venv
+            source venv/bin/activate
+            pip install -U pip setuptools wheel
+            pip install -e '.[tests]' -c constraints.txt
+      - save_cache:
+          key: pip-deps-{{ checksum "constraints.txt" }}-{{ checksum "setup.py" }}-{{ arch }}
+          paths:
+            - ./venv
+      - run:
+          name: Run Linters
+          command: |
+            set -euxo pipefail
+            source venv/bin/activate
+            flake8 autodh autodh/tests
+            bandit -r --ini setup.cfg
+            # TODO -- make these pass on the forked repo
+            # isort --recursive --check-only --diff --settings-path setup.cfg autodh autodh/tests
+            # mypy --config-file setup.cfg
+      - run:
+          name: Run Pytest
+          command: |
+            set -euxo pipefail
+            source venv/bin/activate
+            pytest -c .circleci/resources/pytest_build_config.ini
```

Comment:
There is a bug in pytest related to specifying the the pytest config. Do the following instead:
```
cp -f .circleci/resources/pytest_build_config.ini pytest.ini
pytest
```

Reply:
ah that was the move. Got it. I was wondering why following pytest.ini was causing issues.